packages/jest-runtime/src/index.ts

```diff
@@ -647,11 +647,12 @@ class Runtime {
           return module;
         }
       }
-      throw new Error(
+      const err = Resolver.createModuleNotFoundError(
```

Comment:
Looks like this is a breaking change, since we allow to provide a custom [`resolver`](https://jestjs.io/docs/en/configuration#resolver-string) and don't expect it to have such static method. Could be avoided by exporting `ModuleNotFoundError` from `jest-resolve` and importing it directly here.

Reply:
Good point, will refactor it ASAP